.github/workflows/mainline-protection.yaml

```diff
@@ -8,6 +8,7 @@ on:
   pull_request:
     branches:
       - mainline
+      - smart-mainline
```

Comment:
I would create a separate file for `smart-mainline` since the name field should be different. 

Please create PR against **smart-develop** branch, not **smart-mainline** branch

Reply:
Put this new file in the smart-branch -- https://github.com/awslabs/fhir-works-on-aws-deployment/pull/204